### DISCUSSION TITLE
custom.js:124 Uncaught SyntaxError: Invalid or unexpected token

### DISCUSSION THREAD

**Discussion by isaacdarcilla** [livewire/livewire#3189] [7. Bugs (archived)]
### Discussed in https://github.com/livewire/livewire/discussions/3177

<div type='discussions-op-text'>

<sup>Originally posted by **isaacdarcilla** June 24, 2021</sup>
I have a weird error says **Uncaught SyntaxError: Invalid or unexpected token** at line _custom.js:124_

![image](https://user-images.githubusercontent.com/22732118/123213262-60721800-d4f8-11eb-8533-298c69208526.png)

![image](https://user-images.githubusercontent.com/22732118/123213307-6f58ca80-d4f8-11eb-9f71-288f506d1f24.png)

Did I miss something?</div>

**Comment by pwronline** [comment]
Try this

`@this.birth_date = e.target.value;`

**Comment by isaacdarcilla** [comment]
Not working.
